src/refactoring/models/cart.ts

```diff
@@ -1,28 +1,81 @@
 import { CartItem, Coupon } from "../../types";
 
+//* 장바구니 품목별 계산
 export const calculateItemTotal = (item: CartItem) => {
-  return 0;
+  const discount = getMaxApplicableDiscount(item);
+
+  return item.quantity * item.product.price * (1 - discount);
 };
 
+//* 적용가능한 최대할인율 가져오기
```

Comment:
```suggestion
/**
 * @description 장바구니 품목별 계산 
 */
```

주석 사용하실 때 요런식으로 처리하면 다른 파일에서 마우스만 올려도 맥락 파악할 때 유용합니다b

Reply:
오.. 좋네요! 감사합니다 화이팅!!